src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -161,24 +161,16 @@ public string SelectedLanguage
             }
             set
             {
-                selectedLanguage = value;
-                RaisePropertyChanged(nameof(SelectedLanguage));
-            }
-        }
-
-        /// <summary>
-        /// Controls the Selected option in Node Font Size ComboBox
-        /// </summary>
-        public string SelectedFontSize
```

Comment:
So there is no font-size setting in the preferences panel?

Reply:
Hi @reddyashish We removed such a setting. There was a font size setting when we started Preferences Panel project and recently we confirmed with customers that such a setting would come with no major value. Customers would rely on workspace zoom level after all the improvements we did. So I had a PR to remove the UI, now in this PR I am removing the underneath binding code as well..